sdk/src/androidTest/java/com/mapbox/maps/MapboxMapIntegrationTest.kt

```diff
@@ -47,4 +47,10 @@ class MapboxMapIntegrationTest {
   fun testMapViewIsRenderingSupported() {
     assertEquals(true, MapView.isRenderingSupported())
   }
+
+  @UiThreadTest
+  @Test
+  fun testMapViewIsTerrainRenderingSupported() {
+    assertEquals(true, MapView.isTerrainRenderingSupported())
```

Comment:
how to check `false` state?

Reply:
guess we should select some very bad device, idk if we need to invest time trying to find bad device on existed device farms. Perhaps we can find one on our own device farm one day 😄

Reply:
so it might be fluky 😄